os/fs/driver/mtd/smart.c

```diff
@@ -2163,6 +2163,44 @@ static int smart_scan(FAR struct smart_struct_s *dev)
 #endif
 			}
 
+			fdbg("Duplicate Sector winner=%d, loser=%d\n", winner, loser);
```

Comment:
We've used the terms winner and loser in our code internally,
However, I'd request you to use other more proper terms for the debug statements please.
Somebody else might not be able to figure out what competition is happening here.

How about "active sector" and "inactive sector? instead of winner and loser respectively?

Reply:
I changed